deploy/playbooks/deploy.yml

```diff
@@ -16,3 +16,4 @@
      repos_root: "/opt/repos"
      branch: "master"
      development_server: false
+     app_user: "nginx"
```

Comment:
this reads a bit odd, because the "app", as I understand it, is the Python application. And I don't think that the 'nginx' user should be the owner there, unless this implies just using the nginx user for everything related to the app: from logs to python code and static files?